components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/internal/beans/LiveSyncConfig.java

```diff
@@ -27,6 +27,8 @@
     private boolean enabled;
     private String advertisedHost = "";
     private int advertisedPort = 0;
+    private String username;
+    private String password;
```

Comment:
Maybe we should consider storing the password as char[], since Strings are immutable?

Reply:
Created an issue for this #708

Reply:
Noted.